docs/csharp/programming-guide/interfaces/index.md

```diff
@@ -44,7 +44,7 @@ A base class can also implement interface members by using virtual members. In t
 
 An interface has the following properties:
 
-- An interface is typically like an abstract base class with only abstract members. Any class or struct that implements the interface must implement all its members. Optionally, an interface may define default implementations for some or all of its members.
+- An interface is typically like an abstract base class with only abstract members. Any class or struct that implements the interface must implement all its members. Optionally, an interface may define default implementations for some or all of its members, for more information see [default interface methods](../tutorials/default-interface-methods-versions).
```

Comment:
In the "related sections" this should link to the [interface](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/keywords/interface) language reference page. That also has a thorough explanation of default interface methods.